src/Components/VitalsMonitor/VitalsMonitorAssetPopover.tsx

```diff
@@ -0,0 +1,74 @@
+import CareIcon from "../../CAREUI/icons/CareIcon";
+import { AssetData } from "../Assets/AssetTypes";
+import ButtonV2 from "../Common/components/ButtonV2";
+import { navigate } from "raviger";
+import { useTranslation } from "react-i18next";
+import { Popover, Transition } from "@headlessui/react";
+import { Fragment } from "react";
+
+interface VitalsMonitorAssetPopoverProps {
+  asset?: AssetData;
+}
+
+const VitalsMonitorAssetPopover = ({
+  asset,
+}: VitalsMonitorAssetPopoverProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="max-w-full">
+      <Popover className="relative">
+        <Popover.Button>
+          <CareIcon className="care-l-info-circle cursor-pointer text-sm text-gray-500 hover:text-white md:text-base" />
+        </Popover.Button>
+        <Transition
+          as={Fragment}
+          enter="transition ease-out duration-200"
+          enterFrom="opacity-0 translate-y-1"
+          enterTo="opacity-100 translate-y-0"
+          leave="transition ease-in duration-150"
+          leaveFrom="opacity-100 translate-y-0"
+          leaveTo="opacity-0 translate-y-1"
+        >
+          <Popover.Panel className="absolute z-50 mt-2 w-80 translate-x-[-300px] rounded-md bg-white md:w-96 md:-translate-x-full">
+            <div className="flex flex-col gap-2 p-5">
+              <div className="flex gap-2 text-lg font-bold">
+                <CareIcon className="care-l-lungs" />
```

Comment:
We could use the asset's icon based on the asset type like how we have in the asset list/detail page instead of lungs which'd be more appropriate?

Reply:
Got it, working on the changes